spec/models/peep_spec.rb

```diff
@@ -0,0 +1,14 @@
+# require "./app/models/peep"
```

Comment:
Is this commented out because you had problems with TimeCop?  If so, let me know on Slack and I'll provide some tips.